Create 2025-02-13-leveraging-digital-research-infrastructure.md (#965)
open-life-science/open-life-science.github.io: _posts/2025-02-13-leveraging-digital-research-infrastructure.md

---
layout: post
title: Leveraging Digital Research Infrastructure for Environmental DNA Tools in Monitoring Indigenous Territories.
authors:
- mldlopez
image: 
photos:
  name: 
  license: CC BY-SA 4.0
  url: 
---

> 

Author: ```suggestion

*This speedblog, written by [Mark Louie Lopez](https://marklouielopez.weebly.com/), is part of the OLS-9 cohort.*
```

By providing training and resources, this project aims to empower underrepresented researchers and Indigenous communities, fostering a more inclusive and diverse scientific community.
1. **Co-design biodiversity monitoring projects** with Indigenous partners, identifying priority species for eDNA assay development.
2. **Conduct online training sessions** on using DRI for eDNA assay design, targeting equity-seeking researchers and Indigenous scientific staff.
3. **Validate targeted eDNA assays** and apply them to monitor biodiversity in Indigenous lands, providing timely and accurate data to our partners.
4. **Launch a social media campaign** to raise awareness about the role of DRI in advancing eDNA research and supporting equity-seeking groups.

We anticipate engaging over 100 individuals from equity-seeking groups, including Indigenous communities, racialized researchers, and first-generation post-secondary students. 
By measuring the impact of these activities, we aim to demonstrate how DRI can drive innovation and inclusivity in environmental research.

> ## Open Life Science (OLS) Mentoring Program

Author: ```suggestion
## The OLS "Open Seeds" Mentoring Program
```

> The [OLS program](https://we-are-ols.org/openseeds/) is a 16-week personal mentorship and cohort-based training designed to empower individuals to become Open Science ambassadors in their communities. As part of the OLS-9 cohort, I have had the opportunity to:

Author: ```suggestion
The [Open Seeds program](https://we-are-ols.org/openseeds/) is a 16-week personal mentorship and cohort-based training designed to empower individuals to become Open Science ambassadors in their communities. As part of the OLS-9 cohort, I have had the opportunity to:
```

---
layout: post
title: Leveraging Digital Research Infrastructure for Environmental DNA Tools in Monitoring Indigenous Territories.
authors:
- mldlopez
image: https://images.unsplash.com/photo-1511884642898-4c92249e20b6
photos:
  name: Pine Watt
  license: CC BY-SA 4.0
  url: https://unsplash.com/@pinewatt
---

*This speedblog, written by [Mark Louie Lopez](https://marklouielopez.weebly.com/), is part of the OLS-9 cohort.*

As a freshwater biologist and molecular ecologist, my work has always centered on advancing environmental monitoring and conservation science. 
Over the years, I have had the privilege of contributing to this field through research outputs, collaborations, and community engagement. 
One of the most exciting projects I am currently involved in is the development of environmental DNA (eDNA) tools for biodiversity monitoring in Indigenous lands, supported by the [Digital Research Alliance of Canada's DRI EDIA pilot program](https://alliancecan.ca/en/initiatives/dri-investments). 
This project is not only a scientific endeavor but also a collaborative effort to empower Indigenous communities and address pressing environmental challenges.

> ![A photo collage containing four (4) related images, showing members of the Chipewyan Prairie First Nation in Northeastern Alberta, as they collect freshwater fish from Lake Winnefred. In the top left square of the collage, men lined up in a row pull on a fish-filled net. In the top right square, a man stands on ice, overlooking many fish. The bottom left image is of a row of fish on the frozen out lake and on the bottom right is an image of three (3) men tugging on the net.](({% link images/2025-02-louie-blog-photos.jpg %}))

Author: ```suggestion
![A photo collage containing four (4) related images, showing members of the Chipewyan Prairie First Nation in Northeastern Alberta, as they collect freshwater fish from Lake Winnefred. In the top left square of the collage, men lined up in a row pull on a fish-filled net. In the top right square, a man stands on ice, overlooking many fish. The bottom left image is of a row of fish on the frozen out lake and on the bottom right is an image of three (3) men tugging on the net.]({% link images/2025-02-louie-blog-photos.jpg %})
```

> ![A photo collage containing four (4) related images, showing members of the Chipewyan Prairie First Nation in Northeastern Alberta, as they collect freshwater fish from Lake Winnefred. In the top left square of the collage, men lined up in a row pull on a fish-filled net. In the top right square, a man stands on ice, overlooking many fish. The bottom left image is of a row of fish on the frozen out lake and on the bottom right is an image of three (3) men tugging on the net.]({% link images/2025-02-louie-blog-photos.jpg %}){: style="max-width: 70%; max-height: 500px;" }

Author: ```suggestion
![A photo collage containing four (4) related images, showing members of the Chipewyan Prairie First Nation in Northeastern Alberta, as they collect freshwater fish from Lake Winnefred. In the top left square of the collage, men lined up in a row pull on a fish-filled net. In the top right square, a man stands on ice, overlooking many fish. The bottom left image is of a row of fish on the frozen out lake and on the bottom right is an image of three (3) men tugging on the net.]({% link images/2025-02-louie-blog-photos.jpg %}){: style="max-width: 80%; max-height: 600px;" }
```
